index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Harshika's Portfolio</title>
+    <link rel="stylesheet" href="style.css" class="css">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@800&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://kit.fontawesome.com/3ab8daaa6e.css" crossorigin="anonymous">
+    <script src="https://kit.fontawesome.com/3ab8daaa6e.js" crossorigin="anonymous"defer></script>
+</head>
+<body>
+    <div class="mobileinfo">
+        <h1>Welcome</h1>
+          </div>
+          <div class="mobilemenu">
+          <i class="fa-sharp fa-solid fa-bars"></i>  </div>
+          
+    <div class="container header">
+        <div class="flex-menu">
+            <a href="#">Portfolio</a>
+        </div>
+        <div class="flex-menu">
+            <a href="#">About</a>
+        </div>
+        <div class="flex-menu">
+            <a href="#">Contact</a>
+        </div>
+
+    </div>
+    <div class="container main">
+        <div class="flex-info">
+            <h1 class="heading">Hey There, I am Harshika</h1>
+            <h1 class="info">I am a Full Stack Developer</h1>
+            <p>I can help you build interactive and responsive websites. To be sure about the quality of my work, you may view my portfolio.</p>
+        </div>
+    </div>
+  
```

Comment:
- [ ] Kindly wrap your headline section in a `<section>` tag. It represents a generic standalone section of the document.